speakers-cfp_runbook.md

```diff
@@ -37,13 +37,31 @@ best conversations to Austin.
 
 ### Program Committee
 
-TBD
+The program committee is a selection of (at least) 5 people who will be voting on the proposed CFPs for our conference. They will help shape and decide on the agenda and content; this is a fundamental job responsibility; please only volunteer if you understand and are willing to commit to it.
+Committee members _should_ _not_ propose talks/workshops/panels due to the conflict of interest; if they plan on offering content to the event, they should recuse themselves from the program committee.
 
 #### Leader(s)
-- TBD Expected respsonsibilities
+
+* Setup the feedback form LINK (update and/or iterate on it)
+  * Verify the voting is still relevant.
+* Recruit N (at minimum 5) number of committee members; DoD ATX has anywhere between 100-200 CFPs and expects chunks of 15-20 CFPs per member with overlap.
+* Use the spreadsheet LINK to help organize the committee
```

Comment:
This LINK will be a link to the google spreadsheet template.

Reply:
you could add this sort of thing as TODO items at the bottom of the document, perhaps? or permalinks to these comments? since PR comments persist on github but aren't represented well in the living document at any particular point in time

Reply:
Updated with LINK.